server/broadcast.go

```diff
@@ -550,18 +550,33 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string,
 	// [EXPERIMENTAL] send content detection results to callback webhook
 	if DetectionWebhookURL != "" && len(res.Detections) > 0 {
 		glog.V(common.DEBUG).Infof("Got detection result %v", res.Detections)
-		go func(mid core.ManifestID, seqNo uint64, detections []*net.DetectData) {
+		go func(mid core.ManifestID, config core.DetectionConfig, seqNo uint64, detections []*net.DetectData) {
+			type SceneClassificationResult struct {
+				ClassName   string  `json:"name"`
```

Comment:
```suggestion
				Name   string  `json:"name"`
```

^ To match the JSON field name?